manifests/share.pp

```diff
@@ -3,9 +3,9 @@
 # This class setups up a config file to be included into smb.conf
 #
 define winbind::share (
-  $ensure_setting  = present,
-  $settings_hash   = undef,
-  $smb_include_dir = $::winbind::smb_includes_dir,
+  Boolean       $ensure_setting  = lookup('winbind::share::ensure_setting'),
```

Comment:
Do `present` and `absent` really translate to Booleans? Kinda cool but wondering where you found this. :)